src/components/Button/Button.module.scss

```diff
@@ -0,0 +1,62 @@
+@use '../../style/mixins.module.scss';
+
+// sizes
+.small {
+  @include mixins.text-preset(14px);
+  padding: 4px 8px;
+  border-radius: 8px;
+}
+.medium {
+  @include mixins.text-preset(16px);
+  padding: 8px 12px;
+  border-radius: 10px;
+}
+.large {
+  @include mixins.text-preset(18px);
+  padding: 12px 16px;
+  border-radius: 15px;
+}
+// skins
+.none {
+  border: none;
+  background: none;
+}
+.standard {
+  border: 1px solid rgb(0, 110, 255);
+  color: black;
+  background: white;
+}
+.standard:hover {
+  background: rgb(226, 226, 226);
+}
+.simple {
+  border: 1px solid lightgray;
+  color: black;
+  background: white;
+}
+.simple:hover {
+  background: rgb(182, 182, 182);
+}
+.destructive {
+  border: 1px solid red;
+  color: black;
+  background: white;
+}
+.destructive:hover {
+  background: rgb(226, 226, 226);
+}
+// themes
+.active {
+  color: white;
+  background: rgb(0, 110, 255);
+}
+
+// default
+.button {
```

Comment:
if this is the main classname, it should be moved to the top. The state based classnames should be below.